modules/spring-config/src/main/java/org/mule/config/spring/SpringRegistry.java

```diff
@@ -83,31 +104,49 @@ protected void doInitialise() throws InitialisationException
     {
         if (applicationContext instanceof ConfigurableApplicationContext)
         {
-                ((ConfigurableApplicationContext) applicationContext).refresh();
+            ((ConfigurableApplicationContext) applicationContext).refresh();
         }
+
+        initTransformers();
         //This is used to track the Spring context lifecycle since there is no way to confirm the lifecycle phase from the application context
         springContextInitialised.set(true);
     }
 
+    private void initTransformers()
```

Comment:
because there's special logic for them in MuleRegistryHelper. This was already like this, all I did was move the code over. This can definitively be improved but out of scope